sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,7 @@
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				FLUTTER_APPLICATION_PATH = path/to/application;
+				FLUTTER_ROOT = path/to/flutter/repo;
```

Comment:
The Xcode project will presumably be moved around by the user (so stuff like launch screens, icons, entitlements etc..) can be added. Also, the SDK is generated from the flutter/engine repo entirely and does not need flutter/flutter at build time.